packages/product-json-to-csv/test/__snapshots__/map-product-data.spec.js.snap

```diff
@@ -0,0 +1,56 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`::ProductMapping ::mapProperties converts variant image array to strings 1`] = `
+Object {
+  "hasStagedChanges": false,
+  "id": "12345ab-id",
+  "key": "product-key",
+  "state": "my-resolved-state",
+  "variant": Object {
+    "id": 1,
+    "images": "https://example.com/foobar/commer.jpg;https://example-2.com/demo/tools.jpg;image-label",
+    "sku": "A0E200000001YKI",
+  },
+}
+`;
+
+exports[`::ProductMapping ::run should accept a product and return a formatted csv string 1`] = `
+Array [
+  Object {
+    "addedAttr": "",
+    "anotherAddedAttr": "",
+    "article": "sample 089 WHT",
+    "categories": "res-cat-name-1;res-cat-name-2",
+    "categoryOrderHints": "res-cat-name-1:0.015;res-cat-name-2:0.987",
+    "color": "white",
+    "colorFreeDefinition.de": "schwarz-weiß",
+    "colorFreeDefinition.en": "black-white",
+    "createdAt": "2017-01-06T10:54:51.395Z",
+    "designer": "michaelkors",
+    "hasStagedChanges": false,
+    "id": "12345ab-id",
+    "key": "product-key",
+    "lastModifiedAt": "2017-01-06T10:54:51.395Z",
+    "name.de": "Handyhülle",
+    "name.en": "Phone cover",
+    "productType": "resolved-product-type",
+    "published": false,
+    "slug.de": "michaelkors-handyhuelle",
+    "slug.en": "michaelkors-phonecover",
+    "taxCategory": "resolved-tax-name",
+    "variant.id": 1,
+    "variant.images": "https://example.com/foobar/commer.jpg;https://example-2.com/demo/tools.jpg;image-label",
```

Comment:
So the delimiter for images is the same as the delimiter for a image label? 🤔

```js
<img1>;<img1-label>;<img2>;<img3>;<img3-label>
```

Reply:
Good question. The delimiter for image label is set by the client through `multiValDel` option but it defaults to `;`. 

I don't really like it but dont have any better ideas atm. Do you have any thoughts?

Reply:
I assume that the image and its label should be considered as one entry. So I would simply define a new delimiter for the pair of them, and use e.g. `;` for delimiting images.

```
<img1>|<img1-label>;<img2>;<img3>|<img3-label>
```

Does it make sense?

Reply:
Sure does, thx!